experiments/kripke/experiment.py

```diff
@@ -39,102 +39,72 @@ class Kripke(
 
     maintainers("pearce8")
 
-    def compute_applications_section(self):
-        # TODO: Replace with conflicts clause
-        scaling_modes = {
-            "strong": self.spec.satisfies("+strong"),
-            "weak": self.spec.satisfies("+weak"),
-            "throughput": self.spec.satisfies("+throughput"),
-            "single_node": self.spec.satisfies("+single_node"),
-        }
-
-        scaling_mode_enabled = [key for key, value in scaling_modes.items() if value]
-        if len(scaling_mode_enabled) != 1:
-            raise BenchparkError(
-                f"Only one type of scaling per experiment is allowed for application package {self.name}"
-            )
-
-        input_variables = {
+    def initialize_expr_variables(self):
+        expr_vars = {
+            # Number of processes in each dimension
+            "num_procs": {"npx": 2, "npy": 2, "npz": 1},
+            # Number of zones in each dimension, per process
+            "problem_sizes": {"nzx": 64, "nzy": 64, "nzz": 32},
```

Comment:
this would be `x`, `y`, `z` (`--zones <x>,<y>,<z>`), instead of `nzx`, `nzy`, `nzz`?